### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by luxas**
Similar to https://github.com/racklet/drawio-render-action/pull/1

### CODE REVIEW

**Comment by luxas** [comment]
This PR should also now be ready. I'll squash after approval.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## v1.0.0
```
Comment (twelho) [inline CHANGELOG.md:3]: https://github.com/racklet/render-drawio-action/pull/1 reaches `v0.1.0` while this goes straight to `v1.0.0`?

**Review by twelho — APPROVED** [review]
LGTM :+1: noticed just a slight versioning mismatch with `render-drawio-action`, see comment. Would be nice to pick one format and go with that for both projects.

File: CHANGELOG.md (hunk 1 above)
Comment (luxas) [reply CHANGELOG.md:3]: It should be v1.0.0 in both places, fixed

**Comment by luxas** [comment]
Thanks for the review :+1:! Squashed commits now, will merge.

**Pull request merged**